### PR TITLE
Add example of building Slurm on top of Rocky 8

### PR DESCRIPTION
**Pull request by nick-stroud**
Builds the latest Slurm V6 image on top of HPC Rocky 8 image and deploys a simple cluster using the image. This example is meant to be a demonstrator to show how it is possible to build Slurm. 

Notes:
- This calls ansible directly instead of using Slurm Packer module to avoid complexity of importing external Packer module.
- This does custom ansible install instead of using ansible runner as python upgrade breaks selinux bindings when done from within a virtual environment. 

Tested: 
- Manually run blueprint and test how cluster (`srun hostname`).
- Added integration test that does deploy/destroy

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

**Comment by nick-stroud** [comment]
To Reviewer: I will add documentation and integration test once you have given a first pass approval of the methodology of the blueprint.

File: community/examples/hpc-build-slurm-image.yaml
Comment (tpdownes) [inline community/examples/hpc-build-slurm-image.yaml:41]: I would set `install_ansible: false` explicitly and add a comment

- to avoid breaking upon nay future changes in behavior of startup-script
- allow user to add an ansible runner without breaking script (I bet it would work if it came after your installation)
- generally be explicit about a choice that matters

File: community/examples/hpc-build-slurm-image.yaml
Comment (tpdownes) [inline community/examples/hpc-build-slurm-image.yaml:52]: Is version of Ansible critical? Here you are installing a very recent copy of Ansible (good) but the rest of our Ansible solutions are tested with 2.11.

If the user were to add Ansible runners for the actual cluster, it would run on the copy of Ansible you are installing here. It might be wise to align with rest of Toolkit even if that mans an older version.

What do you think?

**Review by tpdownes — CHANGES_REQUESTED** [review]

**Comment by nick-stroud** [comment]
I submitted test manually (build 4c971653-eee2-45f3-8d9c-1cb7c2634145) and it is passing.  
I will update triggers once it is submitted.

**Review by tpdownes — APPROVED** [review]

**Pull request merged**
